Makefile

```diff
@@ -322,18 +318,15 @@ create-cluster: ## Create a development Kubernetes cluster on Azure in a KIND ma
 	  apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
 	# Create 2 worker nodes with MachineDeployment.
 	kubectl \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f examples/_out/machinedeployment.yaml
 
-	@echo 'run "kubectl --kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") ..." to work with the kind cluster'
```

Comment:
should we leave a note here about working with the kind cluster so first time users know where to manage the cluster from?

Reply:
yeah, that's a good idea, i'll add another line and please let me know if it sounds good to you.

Reply:
perfect thanks